editor/filesystem_dock.cpp

```diff
@@ -1448,12 +1448,13 @@ void FileSystemDock::_files_list_rmb_select(int p_item, const Vector2 &p_pos) {
 }
 
 void FileSystemDock::_rmb_pressed(const Vector2 &p_pos) {
-	folder_options->clear();
-	folder_options->set_size(Size2(1, 1));
+	file_options->clear();
+	file_options->set_size(Size2(1, 1));
 
-	folder_options->add_item(TTR("New Folder.."), FOLDER_NEW_FOLDER);
-	folder_options->set_position(files->get_global_position() + p_pos);
-	folder_options->popup();
+	file_options->add_item(TTR("New Folder.."), FOLDER_NEW_FOLDER);
```

Comment:
Now that we're using `file_options`, should it be `FILE_NEW_FOLDER`? :) I'm not sure what's the difference between the two.

Reply:
Hm, true. Will fix it now.